opm/autodiff/GeoProps.hpp

```diff
@@ -186,6 +192,11 @@ namespace Opm
                                      const double* perm,
                                      Vector &hTrans);
 
+        template <class Grid>
+        void minPvFillProps_(const Grid &grid,
```

Comment:
I see, did not notice. If I find a tool that automatically prettifies all code in a pleasing manner I'll run it on the entire code base... Until then, imitating the style in the file you are editing is the best approach.